webapp-django/crashstats/supersearch/templates/supersearch/search_results.html

```diff
@@ -24,7 +24,7 @@
         <tbody>
             {% for crash in query.hits %}
             <tr>
-                <td><a href="{{ url('crashstats:report_index', crash.uuid) }}" class="external-link">{{ crash.uuid }}</a></td>
+                <td><em><a href="{{ url('crashstats:report_index', crash.uuid) }}">{{ crash.uuid }}</a></em></td>
```

Comment:
I'm not fan of this ``<em>`` here, I'd rather have those crash ids have the same CSS they have in the signature report page: https://github.com/mozilla/socorro/blob/master/webapp-django/crashstats/signature/static/signature/css/signature_report.less#L51-L56